packages/ui/tsconfig.json

```diff
@@ -19,5 +19,5 @@
     "allowJs": true,
     "checkJs": true
   },
-  "include": ["src/**/*.d.ts", "src/**/*.ts", "src/**/*.svelte"]
+  "include": ["src/**/*.d.ts", "src/**/*.ts", "src/**/*.svelte", "../../types/**/*.d.ts"]
```

Comment:
Why do you need the types folder ?

Reply:
it's related to this part of the description 
`
need to import the types in tsconfig.json for humanize-duration library
`

Reply:
Okey np